cms/src/fields/collection.ts

```diff
@@ -32,4 +32,37 @@ export const COLLECTION_FIELDS: Array<CmsField> = [
   PUBLISHED_FIELD,
   HIDDEN_FIELD,
   EXERCISES_FIELD,
+  {
+    label: '🪪 Card',
+    name: 'card',
+    i18n: true,
+    widget: 'object',
+    collapsed: true,
+    required: false,
+    fields: [
+      {
+        label: '🪪 Card description',
+        name: 'description',
+        widget: 'string',
+        i18n: 'duplicate',
+        required: false,
+        hint: 'Description displayed on the card',
+      },
+      {
+        label: '🎨 Background colors',
+        name: 'backgroundColorGradient',
+        i18n: 'duplicate',
+        widget: 'list',
+        required: false,
+        hint: 'Background gradient',
```

Comment:
This needs some more specification. Also there's a color picker widget you can use for the entries of the list.

Reply:
And the hint doesn't add much imo :P

Reply:
Do you have a suggestion for a hint?

Reply:
Tried using the color picker, it works for textColor field but not in a list
<img width="400" alt="Screenshot 2023-06-01 at 11 51 27" src="https://github.com/29ki/29k/assets/7523828/5f4b3e2c-bb7c-4e5d-b032-39f40c50b74f">
<img width="400" alt="Screenshot 2023-06-01 at 11 51 05" src="https://github.com/29ki/29k/assets/7523828/96420daa-b42d-4e03-83e2-1b3d4749c5c5">

they say it is a part of netlify cms and it did work on the textColor
but it also seems buggy when on the bottom of the screen, getting cut:
<img width="350" alt="Screenshot 2023-06-01 at 11 54 46" src="https://github.com/29ki/29k/assets/7523828/8dc6fbb9-0dd4-45bc-81f7-4e0473269e00">

Reply:
Aha, the 'color' schema error seem to happen even without using color widget, investigating 🤔

Reply:
Fixed: [52579b2](https://github.com/29ki/29k/pull/2912/commits/52579b2c8eb363f36b125a0aef9a6591fa656f83)

Reply:
Nice :)